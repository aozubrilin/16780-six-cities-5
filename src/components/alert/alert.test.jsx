import React from "react";
import renderer from "react-test-renderer";
import Alert from "./alert";

const MESSAGE = `Error`;

describe(`Alert snapshot`, () => {
  it(`Should Alert render correctly`, () => {
    const tree = renderer.create(
        <Alert
          message={MESSAGE}
          onClose={() => {}}
        />
    )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});

