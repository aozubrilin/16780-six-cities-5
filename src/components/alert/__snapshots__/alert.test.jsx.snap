// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert snapshot Should Alert render correctly 1`] = `
<div
  className="alert"
>
  <span
    className="closebtn"
    onClick={[Function]}
  >
    Ã—
  </span>
  Error
</div>
`;
